src/event/commands/actions/streaks.ts

```diff
@@ -0,0 +1,64 @@
+import { Message, EmbedBuilder } from 'discord.js'
+
+import zenCountSchema from '../../../schemas/zenCountSchema'
+
+export const streaks = async (
+  message: Message,
+  args: string[],
+): Promise<void> => {
+  if (args.length > 0) {
+    message.reply('Invalid command. Usage: !streaks')
+    return
+  }
+
+  try {
+    const streakEntries = await zenCountSchema
+      .find({})
+      .sort({ streak: -1 })
+      .limit(10)
+      .exec()
```

Comment:
We should remove the users that have a streak of 0
and reset to 0 those who have not said noze in the last 2 hours